### DISCUSSION TITLE
AudibleSync

### DISCUSSION THREAD

**Discussion by kaeptnkrunch** [audiamus/AaxAudioConverter#151] [General]
It is possible to use the **AudibleSync** instead of the outdated **Audible Manager** to get a connection?

**Comment by audiamus** [comment]
I don't think **AudibleSync** retrieves the activation code like **Audible Manager** did. I recommend to use [**Book Lib Connect**](https://github.com/audiamus/BookLibConnect), which also downloads the chapter meta data similar to the Audible Win10 app. Its export is fully compatible with AAX Audio Converter.

**Reply by kaeptnkrunch** [reply]
Nice, sadly Book Lib Connect doesn't support Linux or macOS. But i will look
